xtractmime/_patterns.py

```diff
@@ -0,0 +1,102 @@
+IMAGE_PATTERNS = (
+    (b"\x00\x00\x01\x00", b"\xff\xff\xff\xff", "image/x-icon"),  # A Windows Icon signature
+    (b"\x00\x00\x02\x00", b"\xff\xff\xff\xff", "image/x-icon"),  # A Windows Cursor signature
+    (b"BM", b"\xff\xff", "image/bmp"),  # The string "BM", a BMP signature
+    (b"GIF87a", b"\xff\xff\xff\xff\xff\xff", "image/gif"),  # The string "GIF87a", a GIF signature
+    (b"GIF89a", b"\xff\xff\xff\xff\xff\xff", "image/gif"),  # The string "GIF89a", a GIF signature
+    (
+        b"RIFF\x00\x00\x00\x00WEBPVP",
+        b"\xff\xff\xff\xff\x00\x00\x00\x00\xff\xff\xff\xff\xff\xff",
+        "image/webp",
+    ),  # The string "RIFF" followed by four bytes followed by the string "WEBPVP"
+    (
+        b"\x89PNG\r\n\x1a\n",
+        b"\xff\xff\xff\xff\xff\xff\xff\xff",
+        "image/png",
+    ),  # An error-checking byte followed by the string "PNG" followed by CR LF SUB LF,
+    # the PNG signature
```

Comment:
When comments are for code that either spans multiple lines or is too long, they should go before the code they are for, not inline in the last line of the code.

```suggestion
    # An error-checking byte followed by the string "PNG" followed by CR LF SUB LF,
    # the PNG signature
    (
        b"\x89PNG\r\n\x1a\n",
        b"\xff\xff\xff\xff\xff\xff\xff\xff",
        "image/png",
    ),
```